src/main/java/org/opensearch/security/ssl/util/CertFromFile.java

```diff
@@ -1,3 +1,14 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
```

Comment:
nit: Are License header changes required for this PR? If not, including only the required changes in a PR would be better. When regressions happen, it'd be much easier to revert commits with clear PR/commit scope.

Reply:
License header checks are required by default in the OpenSearch build scripts, seemed like a good opportunity to just do it instead of attempting a work around.